torchvision/ops/decode_png.py

```diff
@@ -0,0 +1,40 @@
+import torch
+from torch import nn, Tensor
+
+
+def decode_png(input):
+    # type: (Tensor) -> Tensor
+    """
+    Decodes a PNG image into a 3 dimensional RGB Tensor.
+    The values of the output tensor are uint8 between 0 and 255.
+
+    Arguments:
+        input (Tensor[1]): a one dimensional int8 tensor containing
+    the raw bytes of the PNG image.
+
+    Returns:
+        output (Tensor[image_width, image_height, 3])
+    """
+    if not isinstance(input, torch.Tensor) or len(input) == 0:
+        raise ValueError("Expected a non empty 1-dimensional tensor.")
+
+    if not input.dtype == torch.uint8:
+        raise ValueError("Expected a torch.uint8 tensor.")
+    output = torch.ops.torchvision.decode_png(input)
+    return output
+
+
+class DecodePNG(nn.Module):
```

Comment:
I was not thinking about adding a `nn.Module` for this operator just yet, and was thinking mostly about keeping it functional for now

Reply:
As I am discovering the project. I mostly mimicked the implementation of the other operators. 

If you think it's too soon to have an `nn.Module` for this. sure, we can remove it. 

Out of curiosity and to help me understand better, why this might be premature ? Do you want to wait until we have more feedback from users ?

Reply:
It is not unreasonable to have a `nn.Module` for it. Users could then write their transform pipeline as follows:
```
transforms = Compose([
    DecodePNG(),
    Resize(224),
    RandomHorizontalFlip(0.5)
])
```
which would be nice. But on another hand, chaining `Compose` functions is limiting in a few ways (see https://github.com/pytorch/vision/issues/1406 and related issues for some discussion), so I would prefer to start with just the functional interface.
My reasoning is that it's easier to add a function than to remove it once it's been added (because it would break backwards-compatibility).

Does this make sense?